MsApplicationPkg/SecureBootRecovery/SecureBootRecovery.c

```diff
@@ -11,15 +11,64 @@
 #include <Guid/ImageAuthentication.h>
 #include <Guid/VariableFormat.h>
 #include <Library/BaseLib.h>
+#include <Library/BaseMemoryLib.h>
 
 #include "RecoveryPayload.h"
 
+//
 // 10 seconds in microseconds
+//
 #define STALL_10_SECONDS  10000000
 
+//
+// The size of an EFI_STATUS in bytes
+//
 #define STATUS_SIZE         (sizeof(EFI_STATUS) * 2)
 #define STATUS_STRING_SIZE  (STATUS_SIZE + sizeof(L"\0"))
 
+//

```

Comment:
I suggest moving some of these macros and definitions to a header file.

Reply:
I tried breaking up the macros and definitions and I wasn't a fan of breaking it immediately. I might leave this to boot team to figure out how they want to organize it.

Reply:
Moved the certificate defines to a different header file, like you suggested. With a proper comment I like it.

Reply:
Yes, it makes it easier to focus on implementation in the C file and quickly find data definitions in the header file.